independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -35,140 +56,239 @@
3uMq0T2OfO/5cI9j3xv6vbuZLxX3vZkvse/9nNyPu7IzvqT5S+x7c5+Ath9n+72RL41inwDvocB+XMKXWtL3lvtxRf2l1qIy+3EVz37cF2X249pdjyV9AuX34/rR9+4V/d6cl7BPIOt7tz2+d0PwJbXf2/rezaDvvTDP9646/hL9XBO+FD/3Hqfsx3Xl9gksV/gS+N76ftzD+n5c9Igv6fe4rdjv3W37BKTvjfc4fT+uu5P9uKJ9Al1xv3fqe3eB7w393kPEl2ReqlXy8hLlBsX3noD+pdB+nM6XRP8S7MfRQ39patU49HvDfpzClzTfG/MS8yX5fRxFssT3jhOTy5dmZvTv42Yi33sv5iXmS+bhPa5wn4D1vf+M8hI96XtzXnqA89ID4C8pfInvcZIv4T3uWo/vjXzJHOQUvhT0l+jxPa6U770D8tJfsb/02osmLyFfiv+0A/kS3uMK8qWdgbyk+d7Mly6N8pLbVyl8b9O/hHwJ/aXI977rY9X3vvfYJC8xX9oq+BLnpcj39vClSeF76/3e5fsEcD+uc997fbIfx3xphdyPOwH4UpE+AbMfp/nep6f7ce2YL7UyfQK5+3Hn5X8fx763/ZF5ufc47vfuKbUfF9/jKv1yP+7CRq7vbffjbuY+AdyP43tck/fjFhq+ZPKS4Uvi+7i23I/DPZSF6HsDX1L7vae73b3dNvd7C9+70H5chfsEZF7y9AkE+r3FflxD3Y+bZ7+Pe927Hwe+t94ngPe4JfQSvjQ/dI+DPgHwvdU+gWnrezdzfO9qn+j3Rr4E/d4KX0rzUpfJS43KgNLv3ek9Dv0len7fe1z43oIvge/NfEnzl4aBL6V9Am6/94DmLyFfUvsETGBS+wQoFGX8JTcvfQn+Euel35kv6b73U5yX3P24Kf0eh/7SUdzvbfbjNL4UO1/2Hvde9h5Hf6fc4/Q+AUpLsb+0TvKlS4rxpXuALyn3uEK+98GD4h4HfAm/j4sf86V9nJdcvnQg5Usvo++t5yXwvSEvvWX4Evve5zm+N/cJhPMS+d78fRzs7Sbfx9HeLvre5C8ZvmT3UMD3Hl0r+BL63uX3404o7nvT47wU8L3PD+/Hoe+N+3Ht3P24yex+XJYv3Z72Vbbkftx82I+TeUm/x+F+nO4vpftxvdzvDXnJ2+/9UNS/VPX0CTQsX+J7XJn9OORLS81+HPre7cT37of9OLWvsl5iP+4LfT+uScjK7atM+wTkflzd+N6Ul8L7cTYvSd8b+FKV8xL43qJPQNuPa+PeruFLYd+b+VJP6nvLe1z+fpwJwh3sxy3DvMR9Aov1PgGzh4J86ULkS0leYt+7Rlped+V/0u99fOx7M1/ie5zwvYEvob+EfQLHOftxx/F+nJOXpjx5acbJS6u8vvewnpf2c58A5qVDyJf2/+7pE4gCk9bvHeWlbfm+dxSYiC9t0PdQRngPRfW9H/T1Cch73BkhvrRl4wvge9ODvspAXsI+gTBf0u9xB/ke93PKl+iXBL73E8yX9vA9DvIS+0ul+FLiL32Q5KWPAnnJ+N7gL53u3UPZxP3em5K89Bz73uk97jaHL6X3uHs5L10h+r0f43sc5yXhL60dfSjiS5PW914S86Vv/L73ZMj3TvOSuMcp+3Gd9lXa/TjuE1D246Dfe1D2ez/n8KU1Xr7E/pLlS/3mHtdi35vz0ibcjwvwpZbhS8X6vSt9Yj9OyUuh/bikr9L43kSCsK8S9+MwLwFfWpbbJ9Dj3Y/rqqt9Aguj2gHhe/eB7/1r0PdeHPOlHicvnRbo96afGu7Hgb/k97272PdGvtSU+3HiHmf5EvcvzW0/blLkJY0v5ezH5fQvBXxv7PfmPgHYj6vzflxd9FV2mT4B6PdmvoT7cZiXeqYrwJdK9Xtv6Owex/1L7HuPS9+7BF8K+95HE1+ijKR9HzfhfB83lslLp6p8ie9xE8Jf8vKlOB9l+dIh0+/NfIn7vf3+0l6dL/E9jvkS3OPs8/KlLykufenypQcjvjQ+YPnSp+h7l+BLG9lfutV3j6MH97hAXnoT8pLiL1EM8vpLqe99h+BL4Hu/pN3j9qX+EgEmzktZf+nx1F96vLi/hHyJ/SXOS1ekvjfyJc5LJwJf+m17fI+72n+PY3/pCuwTiO9xFyS+dyXNS4Mtn+9Nb46+98WGL6l9As/HeemrQF8l7MfpfKnJ+3Ga7419AsZf6h8qvB/HfQLIl/J971sU39vfJ9AWfEn3l2pV+mU7/Uvd0vfG/Tj0vfvV/bglnJewTwD9pfB+XFXdjxN8Sb3H1bO+t94nAPtx6T2u39PvbfsE9LxUfD8O+wRK7sfl+94ND1/S+70nPX0CVfa9hyRfOkbvq8R+78v0/Th63v24Zdo9Dn3v8H7cMtyPa9Brd6G/tLw5/XBlLnxpqlO+hL63zUsb2F/y8qUpT14avkr0e2v3uJE4L1l/SfAl8Je0vHRqkXucfZSXwPc2ecntEzik86XdOl/aK/oEOC8J39vwpZVaXhqhPgGZl3J9b85L0vcO3OPoYzzyvTEv0RP3uIuAL0FeuhN8b+gTuPbOfN97B/Il8pd+jvIS+N47bL/3E7ueQH/Jx5f2MV/64w+KS+49Lv7OLeB7v8t5KZ8vRZCK+wTQ98a8xPe4c3k/jvnSbbHvfdd20SfAfMnkJegTMHxpifCXBgcHbb/36NrUX+rhvd2IL7W1/biA730Y7Mf9O30Cyyxfamb244gvreH9ONEnsNq/Hzeq78dNrvbuobREXgrux13s8qUW+EvF9+MWW76Evre+h3Kf6BN4jPsEyvrebZ/vLfuXmr79uBXJfhztoIi8JPfj+rz7cT3Il8D3VvbjoE8A+BL3L2X24xqiT4D4kn8/rqHzpfB+XMD3/hV8b5UvXWb5kr3HbYV7XJP7vYvxJXqK711wP25a7sdRXkLfm/6/1v0l4EsLg/txzJeadM38BzMdgVIu5rg8AAAAAElFTkSuQmCC) repeat-x, #0D1C2C;
```

Comment:
Is this normal? It doesn't look correct.

Reply:
Hmmm, ignore this one, looks like there's a bug in the GitHub diff rendering.

Reply:
One of the parameters set by Erin was that everything had to be in one file (images, fonts, etc) so there is no outside dependencies. The large blobs are Base64 encoded images (header background) and svg (logo) as well as the Open Sans font.